### DISCUSSION TITLE
How to set the currency format

### DISCUSSION THREAD

**Discussion by nappalm** [iddan/react-spreadsheet#418] [Q&A]
Hi, I have been testing the component and it seems perfect. I just have a question about whether it is possible to assign a specific format to the column when it is not in editable mode. In this case, a currency format.

**Comment by iddan** [comment]
Hey, thanks for the good feedback. You can set a custom `DataViewer` with formatting for currency. There is no currency view support in the component yet but I'm open for PRs 🙏
